<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerStyleCategoriesDialog</class>
 <widget class="QDialog" name="LayerStyleCategoriesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layer Style Categories</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="SpaceCheckListView" name="style_categories_list_view" native="true"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="ok_button">
     <property name="text">
      <string>Ok</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="SemiTristateCheckbox" name="select_all_checkbox">
     <property name="text">
      <string>Select all</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpaceCheckListView</class>
   <extends></extends>
   <header>QgisModelBaker.utils.gui_utils</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SemiTristateCheckbox</class>
   <extends>QWidget</extends>
   <header>QgisModelBaker.utils.gui_utils</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>LayerStyleCategoriesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>LayerStyleCategoriesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
